src/ResourceManager/Websites/documentation/upcoming-breaking-changes.md

```diff
@@ -26,15 +26,11 @@
 -->
 
 # Upcoming Breaking Changes
-## Release 6.0.0 - May 2018
 
-The following cmdlets were affected by this release:
 
-**Get-AzureRmAppServicePlan**
-- Output type changed from ServerFarmWithRichSku to AppServicePlan
+The following cmdlets were affected by this release:
```

Comment:
Looks like you didn't add the upcoming breaking changes?

Reply:
what is the recommendation here? Give we added the upcoming breaking change attribute to the property do we add the same list here as well? Or do we specify changing the return type to PSSite etc..?